docs/static/monitoring/monitoring-mb.asciidoc

```diff
@@ -0,0 +1,210 @@
+[role="xpack"]
+[[monitoring-with-metricbeat]]
+==== Collect {ls} monitoring data with {metricbeat}
+[subs="attributes"]
+++++
+<titleabbrev>{metricbeat} collection</titleabbrev>
+++++
+
+In 7.3 and later, you can use {metricbeat} to collect data about {ls} 
+and ship it to the monitoring cluster, rather than routing it through the 
+production cluster as described in <<monitoring-internal-collection>>.
+
+To learn about monitoring in general, see 
+{stack-ov}/xpack-monitoring.html[Monitoring the {stack}].
+
+//NOTE: The tagged regions are re-used in the Stack Overview.
+
+To collect and ship monitoring data:
+
+. <<configure-shipper,Configure the {ls} instance you want to monitor>>
+. <<configure-metricbeat,Install and configure {metricbeat} to collect monitoring data>>
+
+[float]
+[[configure-shipper]]
+===== Configure the {ls} instance you want to monitor
+
+. Enable the HTTP endpoint to allow external collection of monitoring data:
```

Comment:
There seems to be some confusion here. This paragraph and those below it seem as though they apply to monitoring _Beats_ where there exists a setting called `http.enabled` which creates an HTTP server listening on TCP/5066. 

However, this seems to be a paragraph about configuring monitoring of _Logstash_, where things are different. In Logstash, no such setting as `http.enabled` exists. Instead, a HTTP server is started automatically, listening on TCP/9600.

Below I've included a screenshot just verifying that this does seem to appear in a section on monitoring Logstash, which shouldn't be the case:

<img width="1459" alt="ls" src="https://user-images.githubusercontent.com/111616/64321771-c783ad00-cfb0-11e9-8536-970e6d87cc83.png">

I'm not sure where this confusion was sourced from, but instead of doing a review of what's here, would it be helpful for us to write down the steps as we see them and then we can start over on this? LMK. Thanks!

Reply:
@cachedout  Absolutely! I started with Beats as a baseline and made changes, so your assessment makes sense.  I am working to get my local environment set up for testing in the meantime. Starting at a high level view would certainly be the best approach, and I will very much appreciate your help.

Reply:
OK. Could you please create a GH issue outlining the areas that I can help with and assign it to me? I'm happy to help!